apps/command-palette-wp-admin/bin/build-languages.js

```diff
@@ -0,0 +1,230 @@
+	return json;
+}
+
+// Request and write language files
+async function downloadLanguages( languageRevisions ) {
+	return await Promise.all(
+		langSlugs.map( async ( langSlug ) => {
+			const filename = `${ langSlug }-v1.1.json`;
+
+			const output = `${ TEMP_PATH }/${ filename }`;
+			const translationUrl = `${ LANGUAGES_BASE_URL }/${ filename }`;
+
+			console.log( `Downloading ${ filename }...` );
+
+			const response = await fetch( translationUrl );
+			if ( response.status !== 200 ) {
+				// Script should exit with an error if any of the
+				// translation download jobs for a language included
+				// in language revisions file fails.
+				// Failed downloads for languages that are not
+				// included in language revisions file could be skipped
+				// without interrupting the script.
+				if ( langSlug in languageRevisions ) {
+					throw new Error( `Failed to download translations for "${ langSlug }".` );
+				}
+				return { langSlug, failed: true };
+			}
+
+			const json = await response.json();
+			await writeFile( output, JSON.stringify( json ) );
+
+			console.log( `Downloading ${ filename } complete.` );
+
+			return { langSlug, languageTranslations: json };
+		} )
+	);
+}
+
+// Split language translations
+function buildLanguages( downloadedLanguages, languageRevisions ) {
+	console.log( 'Building languages...' );
+
+	const successfullyDownloadedLanguages = downloadedLanguages.filter( ( { failed } ) => ! failed );
+	const unsuccessfullyDownloadedLanguages = downloadedLanguages.filter( ( { failed } ) => failed );
+
+	if ( fs.existsSync( CALYPSO_STRINGS ) ) {
+		const { translations } = parse( fs.readFileSync( CALYPSO_STRINGS ) );
+		const translationsFlatten = _.reduce(
+			translations,
+			( result, contextTranslations, context ) => {
+				const mappedTranslations = _.mapKeys( contextTranslations, ( value, key ) => {
+					let mappedKey = key.replace( /\\u([0-9a-fA-F]{4})/g, ( match, matchedGroup ) =>
+						String.fromCharCode( parseInt( matchedGroup, 16 ) )
+					);
+
+					if ( context ) {
+						mappedKey = context + String.fromCharCode( 4 ) + mappedKey;
+					}
+
+					return mappedKey;
+				} );
+
+				return _.merge( result, mappedTranslations );
+			},
+			{}
+		);
+
+		const translationsByRef = Object.keys( translationsFlatten ).reduce( ( acc, key ) => {
+			const originalRef = translationsFlatten[ key ].comments.reference;
+
+			if ( ! originalRef ) {
+				return acc;
+			}
+
+			const refs = originalRef.split( '\n' ).map( ( ref ) => ref.replace( /:\d+/, '' ) );
+
+			refs.forEach( ( ref ) => {
+				if ( typeof acc[ ref ] === 'undefined' ) {
+					acc[ ref ] = [];
+				}
+
+				acc[ ref ].push( key );
+			} );
+			return acc;
+		}, {} );
+
+		// CHUNKS_MAP_PATTERN is relative to the project root, while require is relative to current dir. Hence the `../`
+		const chunksMap = require( '../' + CHUNKS_MAP_PATTERN );
+
+		// Get only the strings that are relevant to the modules for the odyssey-stats app
```

Comment:
Minor copy-paste leak.
```suggestion
		// Get only the strings that are relevant to the modules for the app
```